src/VirtoCommerce.Platform.Core/Assets/IBlobStorageProvider.cs

```diff
@@ -17,48 +18,82 @@ public interface IBlobStorageProvider
         Task<BlobEntrySearchResult> SearchAsync(string folderUrl, string keyword);
 
         /// <summary>
-        /// Get blog info by url
+        /// Get blob info by URL
         /// </summary>
         /// <param name="blobUrl"></param>
         /// <returns></returns>
         Task<BlobInfo> GetBlobInfoAsync(string blobUrl);
 
         /// <summary>
-        /// Create blob folder in specified provider
+        /// Create blob folder
         /// </summary>
         /// <param name="folder"></param>
         Task CreateFolderAsync(BlobFolder folder);
 
         /// <summary>
         /// Open blob for reading
         /// </summary>
-        /// <param name="blobUrl">Realative or absolute blob url (tmp/blob.txt) </param>
+        /// <param name="blobUrl">Relative or absolute blob URL (tmp/blob.txt) </param>
         /// <returns></returns>
+        [Obsolete("OpenReadAsync")]
```

Comment:
I think we shouldn't mark sync methods as obsolete

Reply:
mark removed